torch/__init__.py

```diff
@@ -1898,14 +1901,24 @@ def fn(model: Callable):
                            disable=disable)
         return fn
 
+    def rewrap(model):
+        return functools.partial(
+            compile,
+            fullgraph=fullgraph,
+            dynamic=dynamic,
+            backend=backend,
+            mode=mode,
+            options=options,
+            disable=disable)(model)
```

Comment:
I'm a bit confused by the usage of partial here, perhaps you want:
```suggestion
    rewrap = functools.partial(
            compile,
            fullgraph=fullgraph,
            dynamic=dynamic,
            backend=backend,
            mode=mode,
            options=options,
            disable=disable)
```